<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vip_settings</class>
 <widget class="QDialog" name="vip_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>155</width>
     <height>130</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>155</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>会员官网</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>155</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>会员专区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>65</y>
      <width>155</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>热门功能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>95</y>
      <width>155</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>升级为会员</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>72</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
